doc/architecture/decisions/0015-handle-nomis-users-without-inheritance-in-cas2.md

```diff
@@ -0,0 +1,201 @@
+# 15. Handle Nomis users without inheritance in CAS2
+
+Date: 2023-09-13
+
+## Status
+
+Draft
+
+## Context
+
+The users of the CAS2 service need to authenticate using Nomis credentials as
+they are Prison Offender Managers (POMs). This is in contrast to CAS1 and CAS3,
+whose users log in with their Delius credentials.
+
+At present the CAS API's concept of a `User` is hard-wired to Delius. Its
+properties include:
+
+- `deliusUsername`
+- `deliusStaffCode`
+- `deliusStaffIdentifier`
+- `probationRegion`
```

Comment:
Yes: username (String) and staff id (Long) are common to both Nomis and Delius. Probation region is obviously not meaningful for Prison staff. Perhaps we need to learn from more identity providers before we move to an abstraction of `AuthenticatedUser`? We will soon be extending our auth to external NACRO users so that will give us more information...